tests/pytorch/test_pytorch_autolog.py

```diff
@@ -406,3 +406,4 @@ def test_autolog_registering_model():
 
         registered_model = MlflowClient().get_registered_model(registered_model_name)
         assert registered_model.name == registered_model_name
+
```

Comment:
remove this

Reply:
just for triggerring cross test, I will remove it before merging